cmd/statusd/main.go

```diff
@@ -150,6 +155,50 @@ func main() {
 		return
 	}
 
+	if config.PushNotificationServerConfig.Enabled {
+		if config.NodeKey == "" {
+			logger.Error("node key needs to be set if running a push notification server")
+			return
```

Comment:
Why does it need a `NodeKey` set though? Why would the generated one be bad? Because it would change at restart?

Reply:
Yes, node key should be stable, otherwise client will not be able to communicate with the server anymore

Reply:
Makes sense.